packages/tasks_panel/src/misc/e2e-config.ts

```diff
@@ -0,0 +1,220 @@
+import { GlobPattern, TaskDefinition, Uri, extensions, workspace } from "vscode";
+import { BasToolkit } from "@sap-devx/app-studio-toolkit-types";
+import {
+  Dictionary,
+  compact,
+  concat,
+  extend,
+  find,
+  includes,
+  isEmpty,
+  isMatch,
+  last,
+  reduce,
+  size,
+  split,
+} from "lodash";
+import { getUniqueTaskLabel, updateTasksConfiguration } from "../../src/utils/task-serializer";
+import { DEFAULT_TARGET, cfGetConfigFileField, cfGetTargets } from "@sap/cf-tools";
+import { getLogger } from "../../src/logger/logger-wrapper";
+import * as path from "path";
```

Comment:
subjective opinion: avoid using `import * as ...`

Reply:
Maybe not fully subjective, it could impact tree-shaking algorithms for bundling

Reply:
![image](https://github.com/SAP/task-explorer/assets/4233843/75cf02f5-509d-46da-b522-dfa60dabf426)
https://web.dev/articles/reduce-javascript-payloads-with-tree-shaking